doc/dev/osd_internals/manifest.rst

```diff
@@ -155,6 +275,9 @@ The type enum reflects three possible states an object can be in:
 
 TODO: apparently we specify the offset twice, with different widths
 
+-->  We need two offsets, The former is the source offset and the latter
+is the target offset.
```

Comment:
Ah, the chunk_info_t::offset is in the chunk object.  Ok, you can just remove that TODO.